webpack.embedding-sdk.config.js

```diff
@@ -125,17 +124,14 @@ module.exports = env => {
       "react/jsx-runtime": "react/jsx-runtime",
     },
 
-    optimization: !isDevMode
-      ? {
-          minimizer: [
-            new TerserPlugin({
-              minify: TerserPlugin.swcMinify,
-              parallel: true,
-              test: /\.(tsx?|jsx?)($|\?)/i,
-            }),
-          ],
-        }
-      : undefined,
+    optimization: {
+      // The default `moduleIds: 'named'` setting breaks Cypress tests when `development` mode is enabled,
+      // so we use a different value instead
+      moduleIds: isDevMode ? "natural" : undefined,
+
+      minimize: !isDevMode,
+      minimizer: mainConfig.optimization.minimizer,
```

Comment:
The `mainConfig` has TerserPlugin with the exact same config, so we can just re-use it and toggle the `minimize` field instead